modules/schemas/src/main/resources/contexts/nexus/core/schema/v0.1.0.json

```diff
@@ -1,6 +1,5 @@
 {
   "@context": [
-    "{{base}}/contexts/nexus/core/resource/v0.1.0",
```

Comment:
I would suggest to put back the standard context (which have the resource context )

```json
{
  "@context": [
    "{{base}}/contexts/nexus/core/standards/v0.1.0",
    "{{base}}/contexts/nexus/core/shacl20170720/v0.1.0"
  ]
}
```

Reply:
`shacl20170720` imports `standards` so I'm not sure to understand the point.

Reply:
In that case no need to use nexus/core/standards/v0.1.0 here.